scripts/build-release.sh

```diff
@@ -8,6 +8,7 @@ set -e
 source ./scripts/test_lib.sh
 
 VERSION=$1
+CURRENT_BRANCH=${CURRENT_BRANCH:-0}
```

Comment:
Please add a comment what values does the parameter takes. 
It's not obvious that it takes 0 & 1 and what 1 means.

Reply:
Since build-release.sh just transparently passes the parameter to build-binary.sh, so I added comment in build-binary.sh like below,
```

# When ${CURRENT_BRANCH} is 0, the scripts doesn't clone the remote
# repository at all, nor checkout the branch; instead, it just builds
# the binaries in current branch in local existing directory.
# Otherwise if ${CURRENT_BRANCH} is 1, it clones the remote repository
# firstly, then checkout the target tagged version, and finally build
# binaries.
```